data/text/common_1.asm

```diff
@@ -153,7 +153,7 @@ _CardFolderOpenText::
 	text "CARD FOLDER open.@"
 	text_end
 
-_OakTimeText1::
+_OakTimeWokeUpText::
```

Comment:
_OakEllipsesText

Reply:
Why? The full text is:
```
	text "<……><……><……><……><……><……>"
	line "<……><……><……><……><……><……>"

	para "Zzz… Hm? Wha…?"
	line "You woke me up!"

	para "Will you check the"
	line "clock for me?"
	prompt
```

Reply:
You're right, I just saw the first part.